blackbox/docs/administration/ssl.txt

```diff
@@ -195,10 +239,13 @@ Generate a Self-Signed Certificate
 
 If you don't get your certificate signed from one of the official CAs,
 you might want to create your own CA with a self-signed certificate.
+The common name (CN) should overlap with the CN of the server key
```

Comment:
You're right. Should be clarified. What I mean is that `*.crate.io` overlaps `ssl.crate.io`.